eng/Version.Details.xml

```diff
@@ -30,12 +30,12 @@
     </Dependency>
   </ProductDependencies>
   <ToolsetDependencies>
-    <Dependency Name="Microsoft.DotNet.Arcade.Sdk" Version="8.0.0-beta.23262.5">
+    <Dependency Name="Microsoft.DotNet.Arcade.Sdk" Version="8.0.0-beta.23360.1">
       <Uri>https://github.com/dotnet/arcade</Uri>
-      <Sha>1aff4eb33aa7cbf26ccd9fc43c17cb609a14dad4</Sha>
+      <Sha>226c103b56a979243f08032e6cddb4deb66ea54b</Sha>
       <SourceBuild RepoName="arcade" ManagedOnly="true" />
     </Dependency>
-    <Dependency Name="Microsoft.Net.Compilers.Toolset" Version="4.6.0-1.23107.10">
+    <Dependency Name="Microsoft.Net.Compilers.Toolset" Version="4.7.0-3.23315.4">
```

Comment:
@MichaelSimons Do we have something misconfigured or is it expected that some entries of the `Version.details.xml` don't get autoupdated?

Reply:
I'm not sure what the intent is here.  A Versions.Details.xml file will only get updated 1 or 2 ways automatically.

1. A darc dependency exists for the dependency
2. A dependency is marked as a coherent dependency and that dependency has a darc subscription.

Because this is an update flow PR for arcade, the compiler dependency would have to be marked as a coherent dependency on arcade if you expect it to be updated when arcade is.

Reply:
I am not sure what needs to be done. I see that this PR got created and was failing because the compilers toolset was not on source build, updating this line in the `Version.Details.xml` removed the issue.

I am assuming that they need to be correlated but I have no idea. If it all look good to you then I am fine.

Reply:
If they need to be correlated then `CoherentParentDependency="Microsoft.DotNet.Arcade.Sdk"` should be added to the dependency.